Bug: The expires information in the Token is not well checked

### What happened?

The expiration information (`expires`) is an integer that represents hours.
The current behavior accepts you to create expires as 0 or negative hours (-2)

See the Functional Tests about it here: https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984

We should restrict it more in this part of the code:
- https://github.com/vmware/repository-service-tuf-api/blob/7b57e1778ec1608cb90d68a1490264e5ba4b3549/repository_service_tuf_api/token.py#L33-L35

and also here
- https://github.com/vmware/repository-service-tuf-api/blob/7b57e1778ec1608cb90d68a1490264e5ba4b3549/repository_service_tuf_api/token.py#L93

### What steps did you take?

1) Login as Admin
2) Generate a token using expires as 0 or -30

### What behavior did you expect?

Accept only expires >1

### Relevant log output

```shell
FAILED tests/functional/tokens/test_generate.py::test_admin_cannot_generate_token_using_api_with_invalid_expires[['read:bootstrap','write:targets']-0] - AssertionError: {"access_token":"***"}
[545](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:546)
assert 200 == 422
[546](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:547)
 +  where 200 = <Response [200]>.status_code
[547](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:548)

FAILED tests/functional/tokens/test_generate.py::test_admin_cannot_generate_token_using_api_with_invalid_expires[['read:bootstrap','write:targets']--5] - AssertionError: {"access_token":"***"}
[548](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:549)
assert 200 == 422
[549](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:550)
 +  where 200 = <Response [200]>.status_code
[550](https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984#step:15:551)
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/docs/source/guide/api/swagger.json
+++ b/docs/source/guide/api/swagger.json
@@ -512,6 +512,7 @@
                     },
                     "expires": {
                         "title": "Expires",
+                        "minimum": 1.0,
                         "type": "integer",
                         "description": "Expiration in hours. Default: 1 hour",
                         "default": 1
@@ -1961,6 +1962,7 @@
                     },
                     "expires": {
                         "title": "Expires",
+                        "minimum": 1.0,
                         "type": "integer",
                         "description": "In hour(s)"
                     }

--- a/repository_service_tuf_api/token.py
+++ b/repository_service_tuf_api/token.py
@@ -31,7 +31,9 @@ class TokenRequestForm:
             ),
         ),
         expires: Optional[int] = Form(
-            default=1, description="Expiration in hours. Default: 1 hour"
+            default=1,
+            description="Expiration in hours. Default: 1 hour",
+            ge=1,
         ),
     ):
         self.username = username
@@ -89,7 +91,7 @@ class TokenRequestPayload(BaseModel):
             SCOPES_NAMES.write_targets.value,
         ]
     ]
-    expires: int = Field(description="In hour(s)")
+    expires: int = Field(description="In hour(s)", ge=1)
 
     class Config:
         example = {

--- a/tests/unit/api/test_token.py
+++ b/tests/unit/api/test_token.py
@@ -194,3 +194,78 @@ class TestPostToken:
         }
         response = test_client.post(url, data=token_data)
         assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+
+    def test_post_with_expires_0(self, test_client):
+        url = "/api/v1/token/"
+        token_data = {
+            "username": "admin",
+            "password": "secret",
+            "scope": ["write:targets"],
+            "expires": 0,
+        }
+        response = test_client.post(url, data=token_data)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert (
+            response.json()["detail"][0]["msg"]
+            == "ensure this value is greater than or equal to 1"
+        )
+
+    def test_post_with_expires_negative(self, test_client):
+        url = "/api/v1/token/"
+        token_data = {
+            "username": "admin",
+            "password": "secret",
+            "scope": ["write:targets"],
+            "expires": -25,
+        }
+        response = test_client.post(url, data=token_data)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert (
+            response.json()["detail"][0]["msg"]
+            == "ensure this value is greater than or equal to 1"
+        )
+
+    def test_post_new_with_expires_0(self, test_client, token_headers):
+        url = "/api/v1/token/new/"
+        payload = {
+            "scopes": ["write:targets"],
+            "expires": 0,
+        }
+        response = test_client.post(url, headers=token_headers, json=payload)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert (
+            response.json()["detail"][0]["msg"]
+            == "ensure this value is greater than or equal to 1"
+        )
+
+    def test_post_new_with_expires_negative(self, test_client, token_headers):
+        url = "/api/v1/token/new/"
+        payload = {
+            "scopes": ["write:targets"],
+            "expires": -31,
+        }
+        response = test_client.post(url, headers=token_headers, json=payload)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert (
+            response.json()["detail"][0]["msg"]
+            == "ensure this value is greater than or equal to 1"
+        )
+
+    def test_post_new_with_expires_missing(self, test_client, token_headers):
+        url = "/api/v1/token/new/"
+        payload = {
+            "scopes": ["write:targets"],
+        }
+        response = test_client.post(url, headers=token_headers, json=payload)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert response.json()["detail"][0]["msg"] == "field required"
+
+    def test_post_new_with_expires_none(self, test_client, token_headers):
+        url = "/api/v1/token/new/"
+        payload = {"scopes": ["write:targets"], "expires": None}
+        response = test_client.post(url, headers=token_headers, json=payload)
+        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
+        assert (
+            response.json()["detail"][0]["msg"]
+            == "none is not an allowed value"
+        )

--- a/tests/unit/conftest.py
+++ b/tests/unit/conftest.py
@@ -33,6 +33,7 @@ def token_headers(test_client):
             "read:settings "
             "read:token "
             "read:tasks "
+            "write:token "
             "delete:targets "
         ),
     }

